diff --git a/scripts/dynsysname b/scripts/dynsysname
index 6906fb224..ce385e046 100755
--- a/scripts/dynsysname
+++ b/scripts/dynsysname
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 P=$1
 
diff --git a/scripts/sysname b/scripts/sysname
index 16f540483..dccd3e677 100755
--- a/scripts/sysname
+++ b/scripts/sysname
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # $Id: sysname,v 1.6 2005/08/09 16:13:16 gquinn Exp $
 # Attempt to guess a canonical system name.
 #   Copyright (C) 1992, 1993, 1994, 1995 Free Software Foundation, Inc.
