diff --git a/Makefile.in b/Makefile.in
index 9d73693d..397002a7 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -367,7 +367,7 @@ src/computeHashSum: $(ComputeHashSum)
 tcl2lua:
 	if [ -d $(srcdir)/pkgs/tcl2lua ]; then                             \
            $(MAKE) -C $(srcdir)/pkgs/tcl2lua  LUA_INC=$(LUA_INCLUDE)       \
-	        TCL_INCLUDE=$(TCL_INCLUDE) TCL_LIBS=$(TCL_LIBS)            \
+	        TCL_INCLUDE=$(TCL_INCLUDE) TCL_LIBS="$(TCL_LIBS)"            \
                 LIB=$(DESTDIR)$(LIB) LIBS=@LIBS@ CC="$(CC)"                \
                 SHARE=$(DESTDIR)$(LIBEXEC) MODE_R=$(MODE_R)                \
                 install;                                                   \
diff --git a/configure b/configure
index 5d9e8ddb..27643553 100755
--- a/configure
+++ b/configure
@@ -4370,18 +4370,6 @@ if test "$ALLOW_TCL_MFILES" = no ; then
 fi
 
 if test "$FAST_TCL_INTERP" = yes ; then
-   OS=$(uname -s)
-   if test "$OS" = Darwin ; then
-     DIR=$(xcrun --show-sdk-path)
-     TCL_INCLUDE="-I${DIR}/usr/include"
-   else
-     for i in "" tcl tcl8.8 tcl8.7 tcl8.6 tcl8.5; do
-        if test -f /usr/include/$i/tcl.h ; then
-	   TCL_INCLUDE="-I/usr/include/$i"
-	   break;
-	fi
-     done
-   fi
 
    if test -z "$TCL_INCLUDE" ; then
        # Extract the first word of "pkg-config", so it can be a program name with args.
@@ -4509,10 +4497,10 @@ return Tcl_CreateInterp ();
   return 0;
 }
 _ACEOF
-for ac_lib in '' tcl tcl8.8 tcl8.7 tcl8.6 tcl8.5
-do
+for ac_lib in '' ; do
   if test -z "$ac_lib"; then
     ac_res="none required"
+    LIBS="$TCL_LIBS $ac_func_search_save_LIBS"
   else
     ac_res=-l$ac_lib
     LIBS="-l$ac_lib $TCL_LIBS $ac_func_search_save_LIBS"
