diff --git a/src/Master.lua b/src/Master.lua
index 6c4c7d41..fb4761f9 100644
--- a/src/Master.lua
+++ b/src/Master.lua
@@ -779,7 +779,7 @@ function M.reload_sticky(self, force)
    local mcp_old  = mcp
    mcp            = MCP
    local reload   = false
-   for i = 1, #stickyA do
+   for i = #stickyA, 1, -1 do
       local entry = stickyA[i]
       local mname = MName:new("load",entry.userName)
       local fn    = mname:fn()
