--- a/intern/cycles/util/transform_inverse.h	2023-01-29 17:59:17.321654038 -0500
+++ b/intern/cycles/util/transform_inverse.h	2023-01-29 17:59:49.249808079 -0500
@@ -12,12 +12,14 @@
 ccl_device_forceinline float3 transform_inverse_cross(const float3 a, const float3 b)
 {
 #ifdef __AVX2__
+#ifdef __KERNEL_SSE2__
   const ssef sse_a = (const __m128 &)a;
   const ssef sse_b = (const __m128 &)b;
   const ssef r = shuffle<1, 2, 0, 3>(
       ssef(_mm_fmsub_ps(sse_a, shuffle<1, 2, 0, 3>(sse_b), shuffle<1, 2, 0, 3>(sse_a) * sse_b)));
   return (const float3 &)r;
 #endif
+#endif
 
   return cross(a, b);
 }
@@ -25,8 +27,10 @@
 ccl_device_forceinline float transform_inverse_dot(const float3 a, const float3 b)
 {
 #ifdef __SSE4_1__
+#ifdef __KERNEL_SSE2__
   return _mm_cvtss_f32(_mm_dp_ps((const __m128 &)a, (const __m128 &)b, 0x7F));
 #endif
+#endif
 
   return dot(a, b);
 }
