diff --git a/src/libstore/local-store.cc b/src/libstore/local-store.cc
index 2556ca485..d4bd0711d 100644
--- a/src/libstore/local-store.cc
+++ b/src/libstore/local-store.cc
@@ -170,7 +170,7 @@ LocalStore::LocalStore(ref<const Config> config)
         else if (!config->readOnly) {
             auto st = stat(config->realStoreDir.get());
 
-            if (st.st_uid != 0 || st.st_gid != gr->gr_gid || (st.st_mode & ~S_IFMT) != perm) {
+            if (st.st_uid != 0 || st.st_gid != gr->gr_gid || (st.st_mode & ~(S_IFMT|7)) != (perm & ~7)) {
                 if (chown(config->realStoreDir.get().c_str(), 0, gr->gr_gid) == -1)
                     throw SysError("changing ownership of path %s", PathFmt(config->realStoreDir.get()));
                 chmod(config->realStoreDir.get(), perm);
