diff --git a/build/build.py b/build/build.py
index 48f4532f..0c28b6fb 100755
--- a/build/build.py
+++ b/build/build.py
@@ -494,8 +494,8 @@ def main():
     f"--output_path={output_path}",
     f"--cpu={wheel_cpu}"])
   print(" ".join(command))
-  shell(command)
-  shell([bazel_path, "shutdown"])
+  with open("bazel_args", 'w') as f:
+    f.write('\n'.join(command[1:]))
 
 
 if __name__ == "__main__":
